### ISSUE TITLE
links to http:

### ISSUE THREAD

**Issue by musicpanda** [thirtybees/thirtybees#885]
A rather common problem when using SSL is mixed content. However, it is quite difficult to search for the guilty code as each file contains a reference to http://opensource.org/licenses/osl-3.0.php in its heading.

So my suggestion is to update all those heading links to https://opensource.org/licenses/osl-3.0.php

**Comment by Dh42** [comment]
Are there actually templates that use hard coded links starting with http other than the comments?

**Comment by musicpanda** [comment]
There certainly are older modules...

**Comment by Traumflug** [comment]
Not that I consider this to be a bug, but I changed validation templates to use all HTTPS, see c83db16f8f76458e42af712644041057497444f8. Currently a whole lot of (41 ea.) modules are waiting for a release due to such small changes, so it's probably a good point time to do this now. As of this commit, each forthcoming release will have HTTPS links only.

----
Regarding finding text, I typically use this:
```
grep --exclude-dir cache --exclude-dir vendor --exclude-dir filemanager --exclude-dir jquery --exclude-dir ace -rn <some search>
```
This can get limited to lines shorter than 500 chars (excluding compiled files):
```
grep --exclude-dir cache --exclude-dir vendor --exclude-dir filemanager --exclude-dir jquery --exclude-dir ace -rn <some search> | grep -v '.\{500\}'
```
And also to exclude certain results:
```
grep --exclude-dir cache --exclude-dir vendor --exclude-dir filemanager --exclude-dir jquery --exclude-dir ace -rn <some search> | grep -v <something to avoid> | grep -v <something else to avoid> | ...
```

**Comment by Traumflug** [comment]
Unless I'm missing something, this issue is solved with bc27b2d. If not, please don't hesitate to re-open it.
